plugins/search-react/src/components/DefaultResultListItem/DefaultResultListItem.test.tsx

```diff
@@ -19,6 +19,8 @@ import { renderInTestApp } from '@backstage/test-utils';
 import FindInPageIcon from '@material-ui/icons/FindInPage';
 import { DefaultResultListItem } from './DefaultResultListItem';
 
+jest.mock('@backstage/canon/src/hooks/useMediaQuery');
```

Comment:
This deep mock will not be available to projects outside of this repo. We have [`mockBreakpoint`](https://github.com/backstage/backstage/blob/d8cf6837ae4d2281c44d89e9b2a06716274e990c/packages/core-components/src/testUtils.ts#L81) in `@backstage/core-components`, perhaps we could add something similar to `@backstage/canon`? (👀 @cdedreuille)

Reply:
Yes agreed that would be a good idea. The more we can make tests easier to build with Canon the better :)